src/KeyVault/Microsoft.Azure.KeyVault/Customized/ObjectIdentifier.cs

```diff
@@ -253,8 +335,16 @@ public CertificateOperationIdentifier(string identifier)
         }
     }
 
+    /// <summary>
+    /// The Key Vault issuer identifier.
+    /// </summary>
     public sealed class IssuerIdentifier : ObjectIdentifier
```

Comment:
>IssuerIdentifier [](start = 24, length = 16)

I will recommend naming this as CertificateIssuerIdentifier as it's a part of certificates collection.